api/egl.xml

```diff
@@ -1069,8 +1069,8 @@
     </enums>
 
     <enums namespace="EGL" start="0x3530" end="0x353F" vendor="MESA" comment="Reserved for Simon Zeni (PR 165)">
-            <enum value="0x3530" name="EGL_GL_TEXTURE_CUBE_MAP_MESA"/>
-                <unused start="0x3531" end="0x353F"/>
```

Comment:
Split  the space changes into separate commit and commit as the first commit of this MR